### PR TITLE
Feature/opinions

### PR DESCRIPTION
**Pull request by JMGarCas**
Users which belong to a voting's census can write anonymous comments on the chat room of that voting.

Make sure to run both makemigrations and migrate of booth.

### CODE REVIEW

**Review by FelixoGudiel — APPROVED** [review]
Works just like intended, the tests are complete and run properly. Overall good job

File: decide/booth/test-selenium.py [hunk 1]
```diff
@@ -171,3 +171,110 @@ def test_voting_list(self):
...
+            name="Google",
+            client_id="test",
+            secret="test",
+        )
+        # Add the current site to the SocialApp's sites
+        app.sites.add(Site.objects.get_current())
+
+        # Opciones de Chrome
+        options = webdriver.ChromeOptions()
+        options.headless = False
```
Comment (TunixR) [inline decide/booth/test-selenium.py]: @Marchabar This needs to be set to True, otherwise the tests won't pass on the github actions

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]

File: decide/booth/test-selenium.py (hunk 1 above)
Comment (Marchabar) [reply decide/booth/test-selenium.py]: Done!

**Review by AntonioRodriguezRuiz — APPROVED** [review]
Nice! All tests pass and the functionality seems to work fine

**Pull request merged**